mcux/mcux-sdk-patches/drivers-fsl_clock.h-Modified-fsl_clock.h-to-match-re.patch

```diff
@@ -0,0 +1,34 @@
+From 3fc3f9671698cc4900ddcf78af402c5e6d3aa841 Mon Sep 17 00:00:00 2001
+From: Daniel DeGrasse <daniel.degrasse@nxp.com>
+Date: Thu, 2 Sep 2021 11:34:18 -0500
+Subject: [PATCH] drivers: fsl_clock.h: Modified fsl_clock.h to match reference
+ manual
+
```

Comment:
@danieldegrasse , would you please submit a PR to update clock driver in mcux-sdk? Thanks a lot.

Reply:
Submitted: https://github.com/NXPmicro/mcux-sdk/pull/49